### ISSUE TITLE
4.1.0.M3 - Could not resolve org.grails.plugins:hibernate5:7.1.0.M3

### ISSUE THREAD

**Issue by codeconsole** [grails/grails-core#11695]
`
grails create-app website
cd website
grails create-domain-class Sample

| Resolving Dependencies. Please wait...

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'website'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve org.grails.plugins:hibernate5:7.1.0.M3.
     Required by:
         project :
      > Could not resolve org.grails.plugins:hibernate5:7.1.0.M3.
         > Could not get resource 'https://oss.jfrog.org/oss-snapshot-local/org/grails/plugins/hibernate5/7.1.0.M3/hibernate5-7.1.0.M3.pom'.
            > Could not GET 'https://oss.jfrog.org/oss-snapshot-local/org/grails/plugins/hibernate5/7.1.0.M3/hibernate5-7.1.0.M3.pom'. Received status code 409 from server: 

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

CONFIGURE FAILED in 20s
| Error Error initializing classpath: Could not GET 'https://oss.jfrog.org/oss-snapshot-local/org/grails/plugins/hibernate5/7.1.0.M3/hibernate5-7.1.0.M3.pom'. Received status code 409 from server:  (Use --stacktrace to see the full trace)

`

**Comment by codeconsole** [comment]
https://repo.grails.org/grails/core/org/grails/plugins/hibernate5/
https://oss.jfrog.org/oss-snapshot-local/org/grails/plugins/

**Comment by puneetbehl** [comment]
@codeconsole Sorry for the inconvenience. Although Grails 4.1.0.M3 is released but due to nature of this release I have to do Grails Framework 4.1.0.M3 release with future BOM versions of a few plugins . 

Right now, Grails Framework 4.1.0.M3 is not ready but once I am finished with other releases it will work fine. In any case, I will keep you posed. 

Thanks!

**Comment by codeconsole** [comment]
Any reason why M1 and M2 worked right away, but M3 does not?  I installed M3 using sdkman.

**Comment by davidkron** [comment]
I tried 4.1.0.M3 with one of our projects and there still seems to be something wrong. I then tried a new project with `grails create-app` with 4.1.0.M3 installed through sdkman and it seems the create-app skeleton is not correct. `build.gradle` contains the dependency `classpath "org.grails.plugins:hibernate5:7.1.0.M2"` instead of `classpath "org.grails.plugins:hibernate5:7.1.0.M3"`.

When I update to the new version I get the following error:
```
| Resolving Dependencies. Please wait...

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'grails_410'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find io.micronaut:micronaut-inject-groovy:.
     Required by:
         project : > org.grails.plugins:hibernate5:7.1.0.M3

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

CONFIGURE FAILED in 2s
| Error Error initializing classpath: Could not find io.micronaut:micronaut-inject-groovy:.
Required by:
    project : > org.grails.plugins:hibernate5:7.1.0.M3 (Use --stacktrace to see the full trace)
```

**Comment by arixmkii** [comment]
@davidkron check this [thread](https://github.com/grails/gorm-hibernate5/issues/251). It also provides a workaround for the issue, which is good enough for the moment.

**Comment by puneetbehl** [comment]
Fixed with Grails 4.1.0.M5
